pkg/virt-handler/cgroup/cgroup.go

```diff
@@ -138,6 +163,23 @@ func getCpuSetPath(manager Manager, cpusetFile string) (string, error) {
 	return cpusetStr, nil
 }
 
+// set cpus "cpusList" on the allowed CPUs. Optionally on a subcgroup of
+// the pods control group (if subcgroup != nil).
+func SetCpuSet(manager Manager, subCgroup string, cpusList []int) error {
+	subSysPath, err := manager.GetBasePathToHostSubsystem("cpuset")
+	if err != nil {
+		return err
+	}
+
+	if subCgroup != "" {
+		subSysPath = filepath.Join(subSysPath, subCgroup)
+	}
```

Comment:
Maybe it makes sense to explicitly check this so we can propagate a clearer error? wdyt?

Reply:
In case the subCgroup does not exist, the error from runc_cgroups.WriteFile will be returned:


// WriteFile writes data to a cgroup file in dir.
// It is supposed to be used for cgroup files only.
func WriteFile(dir, file, data string) error {
        fd, err := OpenFile(dir, file, unix.O_WRONLY)
        if err != nil {
                return err
        }

Whose error format is:

open /path/to/file: error string

This information should be sufficient to diagnose the problem. 
Or do you expect any other particular information which is not printed ?

Reply:
> This information should be sufficient to diagnose the problem.

Let me disagree :)
An `cannot open /my/file/` error message leaves many open questions: what is this file? why are we opening it? in which context? If you wasn't familiar with this area of the code, or even cgroups in general, would you understand where it comes from right away? At least I wouldn't :smiley: 

> Or do you expect any other particular information which is not printed ?

I expect a concrete error message that would help guide whoever's looking at it. Something like: `"Cannot set CPU-set: sub-cgroup <sub-cgroup name> does not exist for cgroup <PATH>: <error>"`.